loki/transformations/transpile/fortran_c.py

```diff
@@ -231,6 +232,31 @@ def transform_subroutine(self, routine, **kwargs):
             header_path = (path/c_kernel.name.lower()).with_suffix('.h')
             Sourcefile.to_file(source=self.codegen(c_kernel, header=True), path=header_path)
 
+
+    def interface_to_import(self, routine, targets):
+        """
+        Convert interface to import.
+        """
+        for call in FindNodes(CallStatement).visit(routine.body):
+            if str(call.name).lower() in as_tuple(targets):
+                call.convert_kwargs_to_args()
+        intfs = FindNodes(Interface).visit(routine.spec)
+        removal_map = {}
+        for i in intfs:
+            for b in i.body:
+                if isinstance(b, Subroutine):
+                    if targets and b.name.lower() in targets:
+                        # Create a new module import with explicitly qualified symbol
+                        modname = f'{b.name}_FC_MOD'
+                        new_symbol = Variable(name=f'{b.name}_FC', scope=routine)
+                        new_import = Import(module=modname, c_import=False, symbols=(new_symbol,))
+                        routine.spec.prepend(new_import)
+                        # Mark current import for removal
+                        removal_map[i] = None
```

Comment:
We might want to use the API properties of `Interface` here:
```suggestion
            for s in i.symbols:
                if targets and s in targets:
                    # Create a new module import with explicitly qualified symbol
                    new_symbol = s.clone(name=f'{s.name}_FC', scope=routine)
                    modname = f'{new_symbol.name}_MOD'
                    new_import = Import(module=modname, c_import=False, symbols=(new_symbol,))
                    routine.spec.prepend(new_import)
                    # Mark current import for removal
                    removal_map[i] = None
```